exercises/practice/difference-of-squares/.meta/config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "authors": [
+    "Atomic-Germ"
+  ],
+  "contributors": [
```

Comment:
This should be empty, it's previous decided that when an exercise is ported from one track to another the list starts blanks.